app/src/main/res/values/strings.xml

```diff
@@ -686,4 +689,9 @@
     <!-- This string is displayed in the message of the notification message shown when the device loses internet connection. -->
     <string name="no_internet_message">Check your device configuration to fix the problem.</string>
 
+
+    <!-- This string is displayed in the message of the notification message shown when the user tries to open a new window and
+      it has already opened the maximum number of allowed windows. -->
+    <string name="max_windows_message">You cannot have more than %1$s windows open at a time. Please close one before opening another.</string>
```

Comment:
You should probably add a comment explaining what `%1$s` expands to.

Reply:
done